go/vt/vtgate/planbuilder/testdata/postprocess_cases.json

```diff
@@ -1,26 +1,4 @@
 [
-  {
-    "comment": "HAVING implicitly references table col",
-    "query": "select user.col1 from user having col2 = 2",
-    "plan": {
-      "QueryType": "SELECT",
-      "Original": "select user.col1 from user having col2 = 2",
-      "Instructions": {
-        "OperatorType": "Route",
-        "Variant": "Scatter",
-        "Keyspace": {
-          "Name": "user",
-          "Sharded": true
```

Comment:
why is this test removed?

Reply:
`col2` is not available in the having clause.  From mysql:

```sql
mysql> select ename from emp having sal > 10;
ERROR 1054 (42S22): Unknown column 'sal' in 'having clause'
```